subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AbstractMultiTestRunner.java

```diff
@@ -316,7 +316,7 @@ private void map(Description source, Description parent) {
             for (Description child : source.getChildren()) {
                 Description mappedChild;
                 if (child.getMethodName() != null) {
-                    mappedChild = Description.createSuiteDescription(String.format("%s [%s](%s)", child.getMethodName(), getDisplayName(), child.getClassName()));
+                    mappedChild = Description.createSuiteDescription(String.format("%s <%s>(%s)", child.getMethodName(), getDisplayName(), child.getClassName()));
```

Comment:
@wolfs Do you see potential risk of this change?

Reply:
Not necessarily. If the tests still run I suppose it is fine.

Reply:
I assume so as this is for human reading.